.github/workflows/ci.yml

```diff
@@ -184,9 +184,23 @@ jobs:
       id-token: write # for docker/login to login to NGINX registry
     secrets: inherit
 
+  build-plus:
+    name: Build Plus images
+    needs: [vars, binary]
+    uses: ./.github/workflows/build.yml
+    with:
+      image: plus
+      platforms: "linux/arm64, linux/amd64"
+    permissions:
+      contents: read # for docker/build-push-action to read repo content
+      security-events: write # for github/codeql-action/upload-sarif to upload SARIF results
+      packages: write # for docker/build-push-action to push to GHCR
+      id-token: write # for docker/login to login to NGINX registry
+    secrets: inherit
+
   functional-tests:
     name: Functional tests
-    needs: [vars, build]
+    needs: [vars, build-plus]
```

Comment:
why not both build and build-plus?

Reply:
Agreed, don't want to run functional tests early if the build-oss step takes too long.